### DISCUSSION TITLE
Game instantly aborts when I load any mod (SOLVED)

### DISCUSSION THREAD

**Discussion by Shard3606** [Raytwo/ARCropolis#485] [Issues]
I recently downloaded arcropolis and it works, except whenever I try to put any mod into the mods file the game crashes on boot up. It happens with any mod and I have tried looking for so many solutions yet nothing helps.

**Comment by AwesomeKirito** [comment]
> I recently downloaded arcropolis and it works, except whenever I try to put any mod into the mods file the game crashes on boot up. It happens with any mod and I have tried looking for so many solutions yet nothing helps.

What happened, how did you fix it
